source/integrate-with-integration-environment/prove-users-identity.html.md.erb

```diff
@@ -80,6 +80,56 @@ Content-Type: application/json
   ]
 }
 ```
+
+## Validating the JWT
+The public keys you must use to validate this JWT are published by OneLogin as [Decentralized Identifiers (DIDs)](https://www.w3.org/TR/did-core/) and can be downloaded from one of the following endpoints:
+
+| Environment | Endpoint                                                         |
+| ----------- | ---------------------------------------------------------------- |
+| Integration | https://identity.integration.account.gov.uk/.well-known/did.json |
+| Production  | https://identity.account.gov.uk/.well-known/did.json             |
+
+You should read this endpoint when validating a document, however to reduce latency we recommend that you incorporate a caching strategy in your service. The endpoint incorporates cache-control headers to facilitate this.
```

Comment:
I'll let other comment on this, but I wonder if we need to expand on our recommendations for caching. We probably want to go into detail as to what the Cache-Control header is and how it should be interpreted!?

Reply:
Added information on cache-control header include example and further information in RFC